### PR TITLE
Hit api branch

### PR DESCRIPTION
**Pull request by Metsanpeitto**
things that I have implemented in this branch : 
- Create a new game with the name of your choice by using the API.
- Implement the "Refresh" button (receiving data from the API and parsing the JSON).
- Implement the form "Submit" button (sending data to the API).
- Use `async` and `await` JavaScript features to consume the API.
- No styling is required.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,3 +1,4 @@
 {
```
Comment (carlos-ssh) [inline .vscode/settings.json:1]: [OPTIONAL]
- Kindly add your .vscode or any other external file of your project into your .gitignore.

File: README.md [hunk 2]
```diff
@@ -12,6 +12,17 @@
...
+### Project requirements
+
+- Read the [Leaderboard API documentation](https://www.notion.so/microverse/Leaderboard-API-service-24c0c3c116974ac49488d4eb0267ade3) to learn how to use this API.
+- Create a new game with the name of your choice by using the API.
+- Implement the "Refresh" button (receiving data from the API and parsing the JSON).
+- Implement the form "Submit" button (sending data to the API).
+- Use `async` and `await` JavaScript features to consume the API.
+- No styling is required.
+
+
```
Comment (carlos-ssh) [inline README.md:24]: [OPTIONAL]
- Keep your professionalism, provide a License at to your README file.

File: README.md [hunk 3]
```diff
@@ -40,7 +51,7 @@ npm install
 ### Usage
 
 ```cmd
-npm test
+npm start
 ```
```
Comment (carlos-ssh) [inline README.md:62]: [OPTIONAL]
- Kindly try adding:  ``npm run build ``

File: src/js/api.js [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+class Api {
+  constructor() {
+    this.baseUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api';
+    this.newGame = 'Maniac Mansion';
+    this.gameId = null;
+    this.scores = [];
+    this.init(); // Calls the API when the page loads for first time and gets scores
+  }
+
+  /**         Checks if a game id is stored in local storage               */
```
Comment (carlos-ssh) [inline src/js/api.js:12]: [OPTIONAL]
- Kindly remove commented lines.

**Review by carlos-ssh — APPROVED** [review]
Hi @Metsanpeitto, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (Metsanpeitto) [reply README.md:24]: Orale! That's a good idea. I will add it Thanks Carlos ! :D

File: src/js/api.js (hunk 4 above)
Comment (Metsanpeitto) [reply src/js/api.js:12]: Thanks for the tip. However, I have to ask you. Is not commenting important? All the comments that I left in my code are directed to help the people who I send to my repo take a look when they have doubts. I realize that I was spending a lot of time helping different students, often with the same issue, so I thought that leave some documentation comments would comply with best practices at the same time it frees me from repeating the same explanation once and again.
Please let me know if there is a better way or if I should do it differently.
Thanks for your time!

**Pull request merged**